standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -134,13 +135,13 @@ object StandardizationJob {
     * @param dataset  A dataset definition
     * @return The updated dataframe reader
     */
-  def getFormatSpecificReader(cmd: CmdConfig, dataset: Dataset)(implicit spark: SparkSession): DataFrameReader = {
+  def getFormatSpecificReader(cmd: CmdConfig, dataset: Dataset, schema: Option[StructType] = None)(implicit spark: SparkSession): DataFrameReader = {
```

Comment:
```suggestion
  def getFormatSpecificReader(cmd: CmdConfig, dataset: Dataset, schemaOpt: Option[StructType] = None)(implicit spark: SparkSession): DataFrameReader = {
```
To distinguish from the other non-optional parameters of the function.

Reply:
thnx